confluent_platform/metadata.csv

```diff
@@ -213,22 +224,25 @@ confluent.kafka.connect.task_error.total_record_errors,gauge,,record,,The number
 confluent.kafka.connect.task_error.total_record_failures,gauge,,record,,The number of record processing failures in this task.,0,confluent_platform,task tot record failures
 confluent.kafka.connect.task_error.total_records_skipped,gauge,,record,,The number of records skipped due to errors.,0,confluent_platform,task tot records skipped
 confluent.kafka.connect.task_error.total_retries,gauge,,operation,,The number of operations retried.,0,confluent_platform,task tot retries
+confluent.kafka.connect.connect_metrics.outgoing_byte_rate,gauge,,byte,,Outgoing byte_rate,0,confluent_platform,incoming byte rate
+confluent.kafka.connect.connect_metrics.incoming_byte_rate,gauge,,byte,,Incoming byte rate,0,confluent_platform,outgoing byte rate
+confluent.kafka.connect.connect_metrics.failed_authentication_rate,gauge,,,,Failed authentication rate,0,confluent_platform,failed auth rate
+confluent.kafka.connect.connect_metrics.successful_authentication_rate,gauge,,,,Successful authentication rate,0,confluent_platform,succ auth rate
+confluent.kafka.connect.connect_metrics.failed_authentication_total,gauge,,,,failed authentication total,0,confluent_platform,fail auth total
+confluent.kafka.connect.connect_metrics.successful_authentication_total,gauge,,,,successful authentication total,0,confluent_platform,succ auth total
+confluent.kafka.connect.connector_metrics.status,gauge,,,,Status of connector,0,confluent_platform,connector status
 confluent.kafka.rest.jetty.connections_active,gauge,,connection,,Total number of active TCP connections (REST).,0,confluent_platform,rest jetty connections active
 confluent.kafka.rest.jetty.connections_opened_rate,gauge,,connection,second,The average rate per second of opened TCP connections (REST).,0,confluent_platform,rest jetty connections opened rate
 confluent.kafka.rest.jetty.connections_closed_rate,gauge,,connection,second,The average rate per second of closed TCP connections (REST).,0,confluent_platform,rest jetty connections closed rate
 confluent.kafka.schema.registry.jetty.connections_active,gauge,,connection,,Total number of active TCP connections (Schema registry).,0,confluent_platform,schema reg jetty connections active
 confluent.kafka.schema.registry.jetty.connections_opened_rate,gauge,,connection,,The average rate per second of opened TCP connections (Schema registry).,0,confluent_platform,schema reg jetty connections opened rate
 confluent.kafka.schema.registry.jetty.connections_closed_rate,gauge,,connection,,The average rate per second of closed TCP connections (Schema registry).,0,confluent_platform,schema reg jetty connections closed rate
-confluent.kafka.rest.jersey.brokers.list.request_error_rate,gauge,,request,second,The average rate of failed brokers list HTTP requests,0,confluent_platform,brokers list err rate
 confluent.kafka.rest.jersey.consumer.assign_v2.request_error_rate,gauge,,request,second,The average rate of failed consumer assign v2 HTTP requests,0,confluent_platform,consumer assign err rate
```

Comment:
Overall, instead of removing metrics from older versions, can you continue supporting these metrics in addition to the new metrics added?

Reply:
Addressed. Metrics were added back in deprecated state (only tested for 5.4)